File: camp/week1/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        leng = len(nums)
        check = nums[0]
        point = 1
        i = 1
        while i < leng:
            if nums[i] != check:
                check = nums[i]
                nums[point], nums[i] = nums[i], nums[point]
                point += 1
            i += 1
        return point 

File: camp/week1/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> temp(nums.begin(),nums.end());
        vector<int> temp2(temp.begin(),temp.end());
        nums.swap(temp2);
        nums.shrink_to_fit();
        return nums.size();
    }
};